gunicorn/arbiter.py

```diff
@@ -501,6 +508,7 @@ def spawn_worker(self):
         pid = os.fork()
         if pid != 0:
             self.WORKERS[pid] = worker
+            self.WORKER_IDS[pid] = worker_id
```

Comment:
This would require adding an instance property to the worker in the base worker constructor, which I did in my initial implementation, but was recommended against above (https://github.com/benoitc/gunicorn/pull/1118#issuecomment-142725348).